modules/placement-driver-api/src/main/java/org/apache/ignite/internal/placementdriver/ReplicaMeta.java

```diff
@@ -19,29 +19,19 @@
 
 import java.io.Serializable;
 import org.apache.ignite.internal.hlc.HybridTimestamp;
+import org.jetbrains.annotations.Nullable;
 
-/**
- * Replica lease meta.
- */
+/** Replica lease meta. */
 public interface ReplicaMeta extends Serializable {
-    /**
-     * Gets a leaseholder node consistent ID.
-     *
-     * @return Leaseholder or {@code null} if nothing holds the lease.
-     */
-    String getLeaseholder();
+    /** Gets a leaseholder node consistent ID (assigned to a node once), {@code null} if nothing holds the lease. */
+    @Nullable String getLeaseholder();
+
+    /** Gets a leaseholder node ID (changes on every node startup), {@code null} if nothing holds the lease. */
+    @Nullable String getLeaseholderId();
 
-    /**
-     * Gets a lease start timestamp.
-     *
-     * @return Lease start timestamp.
-     */
+    /** Gets a lease start timestamp. */
     HybridTimestamp getStartTime();
 
-    /**
-     * Gets a lease expiration timestamp.
-     *
-     * @return Lease expiration timestamp or {@code null} if nothing holds the lease.
-     */
+    /** Gets a lease expiration timestamp. */
     HybridTimestamp getExpirationTime();
```

Comment:
No, we should not use such javadoc pattern.
We have no prohibition on correcting the javadoc style.

Reply:
In that case why did you change that?

Reply:
In any case, I needed to change the documentation for the method, since it cannot return `null`. 
I like short documentation for the code - fewer lines, better looking. 
We have no prohibition on changing the documentation style. 
It's a matter of taste.

Reply:
Ok, up to you here.